MaxVas/HW6/Task1.py

```diff
@@ -0,0 +1,3 @@
+print(f"""Odd numbers divisible by 3: {[i for i in range(1, 10) if i % 3 == 0]}
```

Comment:
output your program:
Odd numbers divisible by 3: [3, 6, 9]
Even numbers divisible by 2: [2, 4, 6, 8]
Not divisible numbers by 2 and 3: [1, 5, 7]

6 is in the both of list. it is not good solution

could try to solve this task again